packages/react-composites/tests/browser/common/defaults.ts

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { Browser, Page } from '@playwright/test';
+
+export type ChatUserType = {
+  userId: string;
+  endpointUrl: string;
+  displayName: string;
+  threadId: string;
+  topic: string;
+  token: string;
+};
+
+export type CallUserType = {
+  userId: string;
+  displayName?: string;
+  groupId?: string;
+  token: string;
+};
+
+export type MeetingUserType = {
+  userId: string;
+  endpointUrl: string;
+  displayName: string;
+  threadId: string;
+  topic: string;
+  groupId?: string;
+  token: string;
+};
+
+export interface WorkerFixture<IdentityType> {
+  serverUrl: string;
+  testBrowser: Browser;
+  users: IdentityType[];
+  pages: Array<Page>;
+}
+
+export const CONNECTION_STRING = process.env.CONNECTION_STRING ?? '';
```

Comment:
nit: I wouldn't mind just throwing here if the connection string isn't set. What's the point of continuing? And throwing here will make the error very obvious.

Reply:
I like that better, will update

Reply:
updated